listenbrainz/webserver/views/user.py

```diff
@@ -139,6 +136,28 @@ def profile(user_name):
     )
 
 
+@user_bp.route("/<user_name>/artists")
+def artists(user_name):
+    """ Show the top artists for the user. These users must have been already
+        calculated using Google BigQuery. If the stats are not present, we
+        redirect to the user page with a message.
+    """
+    user = _get_user(user_name)
+    data = db_stats.get_user_artists(user.id)
```

Comment:
And you're missing error handling. Again. I think we need to add a new tick box to the PR submission form:

[ ] Yes, I think I added enough error checking.

:)

Reply:
Sorry! Will keep in mind going forward. :(

Reply:
@mayhem @paramsingh Hey, I actually thought this was a pretty good idea, so I made a small PR for it. See #311.

Reply:
LOL, funny. :)